src/styles/App.scss

```diff
@@ -99,14 +108,16 @@ a {
   }
 }
 
+$landing-height: 90vh;
```

Comment:
This should probably be at the top. If it's specific to this component, then this component's styling should be broken into a new file